### DISCUSSION TITLE
Remove IcoMoon from Contributing file

### DISCUSSION THREAD

**Discussion by Andre601** [simple-icons/simple-icons#7396] [Ideas]
Under [`2. Extract the Icon from the Logo`](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#2-extract-the-icon-from-the-logo) in the `CONTRIBUTING.md` is IcoMoon mentioned.
While this site works as an online editor, it has severe issues, especially with the unwanted habit of resizing the view box of the SVG, even when it was originally set to 24x24.

Even when you set the grid (size) in the editor to 24x24 will the downloaded SVG usually have a view box of around 600 to 1k+ which certainly isn't wanted.

So, since the editor is doing such a bad job would I recommend removing it from that list (or alternatively add a Notes column to inform about possible drawbacks from those tools/sites such as that the web app of Boxy SVG is extremely slow and unresponsive to a lot of things).

Perhaps some other online tools - mainly for "reviewing" an SVG (See if it looks alright) could be added?
I personally use https://yqnn.github.io/svg-path-editor since it allows me to see the SVG icon through the provided path values and alter it when needed (switch scale, change X/Y position, etc)

Another tool that would mainly help teaching people what stuff does in a path attribute would be https://svg-path-visualizer.netlify.app/ which helped me understand a few of the path attributes.

Either way, I believe IcoMoon isn't really a tool that should be recommended here simply due to the main downsides I listed.

**Comment by sachinraja** [comment]
We can remove IcoMoon then. Feel free to PR an update to `contributing.md`.
